internal/netutil/netutil.go

```diff
@@ -26,3 +30,19 @@ func LocalAddresses() ([]string, error) {
 
 	return result, nil
 }
+
+// LocalProtocol check a condition to use dmsghttp or direct url
+func LocalProtocol() bool {
+	resp, err := http.Get("https://ipinfo.io/country")
```

Comment:
As this `ip.skycoin.com` is used everywhere for location data, I think we should use it here too.

Reply:
I didn't check our service, but I checked `ipinfo.io` by some guys in our mainnet telegram group and work well there.
We can change ours if can confirm that work fine there.

Reply:
I forgot about that issue. I think it's better to just keep it `ipinfo.io`.